app/src/main/java/org/fossasia/openevent/general/utils/CurrencyUtils.kt

```diff
@@ -0,0 +1,24 @@
+package org.fossasia.openevent.general.utils
+
+import timber.log.Timber
+import java.util.*
+
+object CurrencyUtils {
+    private var currencyLocaleMap = HashMap<Currency, Locale>()
+
+    init {
+        for (locale in Locale.getAvailableLocales()) {
+            try {
+                val currency = Currency.getInstance(locale)
+                currencyLocaleMap.put(currency, locale)
+            } catch (e: NullPointerException) {
+                Timber.d(e, "Failed!")
+            }
+        }
+    }
+
+    fun getCurrencySymbol(currencyCode: String?): String? {
```

Comment:
Same method should be used everywhere to convert into the symbol. 
Is this more accurate than the one used before? If yes use this method in the attendee fragment as well.

Reply:
Sure :+1: